### ISSUE TITLE
Have an automated way to 

### ISSUE THREAD

**Issue by claell** [JabRef/jabref#8741]
**Is your suggestion for improvement related to a problem? Please describe.**
When checking for integrity, I get notified that capital letters in a title are not masked in curly brackets. However, I did not find an easy way to correct that (ideally even for all titles during cleanup) automatically.

**Describe the solution you'd like**
Add such an option or, if already there, improve discoverability.

**Additional context**
![grafik](https://user-images.githubusercontent.com/6171135/166162759-a3d270b1-32bf-49c9-9de1-1eb80440da73.png)

**Comment by claell** [comment]
I think that I better understand the problem now. I assume that the message is displayed to indicate that capital letters should be masked if they are wanted to be retrieved during bibliography creation in LaTeX.

Not sure how to best deal with this. DOI and other sources use a title with capitalized letters. Not an expert on this, though.

**Comment by Siedlerchr** [comment]
In BibTeX, you protect the casing with extra curly braces, e.g. {JabRef Developers} for author; otherwise it would get split up. These are also respected by cleanups operations.
You can also use this protect terms feature to create a list of such terms, so they won't be touched by clean ups.

**Comment by claell** [comment]
I found a right click operation for this, also. Should this protection be applied automatically, for example when importing from DOI?

**Comment by Siedlerchr** [comment]
I cannot say for sure, would need to dig again in the code before

**Comment by ThiloteE** [comment]
I think extra curly braces are not necessary when the bibliograhic data is supposed to be used with the OpenOffice/LibreOffice or Microsoft Word integrations. About other Markup tools I don't even know. I would suggest a cleanup action for this instead of to automatically apply curly braces upon import.

**Comment by Siedlerchr** [comment]
Nope, only wrapping the uppercase character is not a good idea. It would be better to wrap the whole word in braces. See this comment for an explanation  https://github.com/JabRef/jabref/issues/419#issuecomment-169689877

**Comment by claell** [comment]
I think that normally, case is not preserved when using .bib files. That is why sometimes words like `{CAD}` or `{AI}` are enclosed in curly brackets (recently had this on a DOI import). The question is, what to do when DOI gives a title like "Approach to Investigate into Topic". Is it wanted to also preserve this?

**Comment by claell** [comment]
From what I found on a short search (https://tex.stackexchange.com/a/140071/194783):
The title should be written in its normal, capitalized form, only masking special words that always needs to be capitalized (like `{CAD}` and `{AI}`). Then the bibliography style used in LaTeX can decide, whether to preserve the normal typed title or to convert non-masked characters to lowercase (more often).

So from that I get that capital letters should normally not be masked, which makes me wonder whether the integrity check result is needed in that case.
